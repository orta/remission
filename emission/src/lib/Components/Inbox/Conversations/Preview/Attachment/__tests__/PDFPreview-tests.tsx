import * as React from "react"
import "react-native"
import * as renderer from "react-test-renderer"

import PDFPreview from "../PDFPreview"

it("renders correctly", () => {
  const tree = renderer.create(<PDFPreview attachment={attachment} />)
  expect(tree).toMatchSnapshot()
})

const attachment = {
  id: "cats",
  file_name: "This is a great PDF telling you all about cats",
}
