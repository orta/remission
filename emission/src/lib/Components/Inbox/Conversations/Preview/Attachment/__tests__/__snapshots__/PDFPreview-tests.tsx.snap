// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
      },
      undefined,
    ]
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View
    style={
      Array [
        Object {
          "flexDirection": "row",
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "borderColor": "#e5e5e5",
            "borderWidth": 1,
            "flexBasis": 0,
            "flexDirection": "row",
            "flexGrow": 1,
            "flexShrink": 1,
          },
          undefined,
        ]
      }
    >
      <Image
        source={1}
        style={
          Array [
            Object {
              "marginBottom": 12,
              "marginLeft": 12,
              "marginTop": 12,
              "resizeMode": "contain",
              "width": 40,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "alignSelf": "center",
              "flexBasis": 0,
              "flexDirection": "column",
              "flexGrow": 1,
              "flexShrink": 1,
              "marginLeft": 25,
            },
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
        >
          This is a great PDF telling you all about cats
        </Text>
      </View>
    </View>
  </View>
</View>
`;
